src/endpoints/metrics.js

```diff
@@ -22,8 +22,8 @@ const getResourceQuery = (resourceType) => {
       },
     }
   }
-  // This is reached by sending a value of "all" which is contrained by openapi
-  return null
+  // This is reached by sending "all" (we cannot get any other values since resourceType is an enum contrained by openapi)
+  return { types: { $regex: ".*" } }
 }
```

Comment:
Same result, but allows us to easily add to the match query in other endpoints, such as adding a filter by group to the query